site/theme/en-US.js

```diff
@@ -34,19 +34,19 @@ module.exports = {
     'app.demo.riddle': 'Open in Riddle',
     'app.home.introduce':
       'A design system for enterprise-level products. Create an efficient and enjoyable work experience.',
-    'app.home.recommend': 'Recommend',
-    'app.home.popularize': 'Popularize',
+    'app.home.recommend': 'Recommended',
+    'app.home.popularize': 'Popular',
     'app.home.design-and-framework': 'Design language and framework',
     'app.home.design-values': 'Design values',
     'app.home.design-values-description':
-      'This is an internal criterion for Ant Design to evaluate good design. Based on the postulate that "everyone pursues pleasure at work", we have added the two values of "Meaningful" and "Growing" on the basis of "Certain" and "Natural" to guide each designer to do better judgment and decision.',
-    'app.home.certainty': 'Certain',
-    'app.home.meaningful': 'Meaningful',
-    'app.home.growth': 'Growing',
-    'app.home.natural': 'Natural',
+      'This is Ant Design\'s internal standard for evaluating design quality. Based on the assumption that "everyone is pursuing happiness at work", we have added the two values of "Meaningfulness" and "Growth" on the basis of "Certainty" and "Naturalness" to guide each designer towards better judgment and decision-making.',
+    'app.home.certainty': 'Certainty',
+    'app.home.meaningful': 'Meaningfulness',
+    'app.home.growth': 'Growth',
+    'app.home.natural': 'Naturalness',
     'app.home.design-guide': 'Guidelines',
     'app.home.components': 'Components',
-    'app.home.detail': 'Detail',
+    'app.home.detail': 'More details',
```

Comment:
Why not `Details`？

Reply:
From what I've seen, usually the link to "view more information" is usually couched with the word "more" in some way -- e.g. wording like:

- "Read more"
- "More info[rmation]"
- "More details"
- "...More"

If you search google for "details" button, this somewhat seems to show that as well:

<img width="1920" alt="Screen Shot 2020-07-16 at 11 54 39 AM" src="https://user-images.githubusercontent.com/463175/87693609-31d1b600-c75b-11ea-98d5-9b73b8b84f4a.png">

It's not 100% always the case of course, as people also use things like:

- Continue reading
- View Details

etc, but I haven't commonly seen just a "details" link by itself -- just my two cents though!

Reply:
ok~